lib/config/presets/internal/index.spec.ts

```diff
@@ -0,0 +1,22 @@
+import * as internal from '.';
+import { validateConfig } from '../../validation';
+import { massageConfig } from '../../massage';
+
+jest.mock('../../../datasource/npm');
+
+const ignoredPresets = ['default:group', 'default:timezone'];
```

Comment:
What are the problems with this presets?

Reply:
They use `{{arg0}}` in their values and fail validation. `default:group` tries to look up a non-existent preset while `default:timezone` is validated against valid timezones.